core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -17,13 +17,13 @@
  */
 
 oppia.controller('FeedbackTab', [
-  '$scope', '$http', '$q', '$uibModal', '$timeout', '$rootScope',
+  '$scope', '$log', '$http', '$q', '$uibModal', '$timeout', '$rootScope',
```

Comment:
Add this in sorted order (alphabetically.)

Reply:
Hey @DubeySandeep 
The other service names are not in any order.
Should I sort them all?

Reply:
That would be great! :)

Reply:
done